documentation/modules/auxiliary/gather/shodan_honeyscore.md

```diff
@@ -1,3 +1,4 @@
+## Introduction
 The `shodan_honeyscore` module utilizes the [Shodan](https://www.shodan.io/) API to determine whether or not a server is a honeypot or not. 
```

Comment:
`whether or not a server is a honeypot or not`

Repeated `or not`. Second instance is redundant.